pkgs/development/python-modules/bsddb3/default.nix

```diff
@@ -31,7 +39,13 @@ buildPythonPackage rec {
   # Path to database need to be set.
   # Somehow the setup.py flag is not propagated.
   #setupPyBuildFlags = [ "--berkeley-db=${pkgs.db}" ];
+
   # We can also use a variable
+  # See : https://github.com/NixOS/nixpkgs/pull/311198#discussion_r1597746759
+  env = {
+    BERKELEYDB_INCDIR = "${lib.getDev pkgs.db}/include";
+    BERKELEYDB_LIBDIR = "${lib.getLib pkgs.db}/lib";
+  };
```

Comment:
I originally meant this to go to the other package (see where I made the comment).
Now I see that both packages have this structure.

I think we should just drop this change for now, maybe in another PR we can add it back.

Reply:
Also, the reason why it wasn't working when you removed `buildInputs` was because you forgot to remove the `preConfigure` below these lines.

Reply:
Seems I did something wrong, thanks for the review, I'll fix this.

Reply:
@TomaSajt  should I remove from `berkeleydb` the change and make it in another PR ?

Reply:
I think you should remove it, but you don't need to create another PR, it's not that important